docs/SingleNodeOpenShift.md

```diff
@@ -0,0 +1,35 @@
+# Single Node OpenShift
+
+## Introduction
+
+This document highlights differences with Machine Config Daemon behaviors in both a highly available (HA) and non-highly-available OpenShift cluster. Single Node OpenShift (SNO) clusters have a single control-plane node and are not considered HA, whereas OpenShift clusters with multiple control-plane nodes are considered HA as control-plane functions can be rescheduled on another node.
+
+## Differences
+
+There are several key differences in how the Machine Config Daemon handles HA and non-HA OpenShift clusters:
+
+### Topology
+
+An annotation (`machineconfiguration.openshift.io/controlPlaneTopology`) is applied to all Node objects to describe the topology of the cluster. HA cluster nodes are annotated with `HighlyAvailable` whereas a non-HA cluster node is annotated with `SingleReplica`. In the absence of this annotation, the Machine Config Daemon defaults to `HighlyAvailable`. This primarily affects the node drain behavior as discussed below.
+
+### Node Drain Behavior
+
+In an HA cluster, a [node drain](./MachineConfigDaemon.md#node-drain) will occur whenever required. However, in a non-HA cluster, a node drain will not occur since there are no other nodes.
```

Comment:
```suggestion
In a non-HA/single node cluster, a node drain will **not** occur since there are no other nodes. This differs from HA clusters, where a [node drain](./MachineConfigDaemon.md#node-drain) will occur whenever required. 
```